server/db/migrations/20231109224923_create-admins-table.js

```diff
@@ -0,0 +1,27 @@
+const tableName = 'admins'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.createTable(tableName, (table) => {
+      // Auto-incrementing non-nullable unsigned integer primary key "id" field
+      table.increments()
+
+      // Simple fields
+      table.string('first_name').notNullable()
+      table.string('last_name').notNullable()
+      table.string('email').notNullable()
+      table.text('password').notNullable()
+      table.boolean('active').defaultTo(true)
+      // Creates an 'created_at' field and 'updated_at' field
+      table.timestamps(true, true)
```

Comment:
I think we should set this to use `table.timestamps(false, true)` instead. That will use Postgres' datetime type, which is a little more flexible.